spec/furniture/marketplace/collecting_payments_system_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+# @see https://github.com/zinc-collective/convene/issues/1576
+describe "Marketplace: Collecting Payments", type: :system do
+  # @see https://github.com/zinc-collective/convene/issues/1622
+  describe "via Stripe" do
+    it "Connecting a Vendor Stripe Account" do
+      space = create(:space, :with_entrance, :with_members)
+      marketplace = create(:marketplace, :with_stripe_utility, room: space.entrance)
+      sign_in(space.members.first, space)
+
+      visit polymorphic_path(marketplace.room.location)
+
+      within("##{dom_id(marketplace, :onboarding)}") do
+        click_on("Manage Marketplace")
+      end
+      click_on("Payment Settings")
+      click_on("View Stripe Account")
+      expect(page).to have_content("Connect to Stripe")
+    end
+  end
+
+  def url_options
```

Comment:
I don't see this method used anywhere obvious. Is it used somehow implicitly? (If so, maybe a comment explaining that might be worth adding.)

Reply:
Potentially copy pasta from https://github.com/zinc-collective/convene/blob/f3c78d6dc4eea3fbb3b7cd011ad0dbb3bbc35e89/spec/furniture/marketplace/buying_products_system_spec.rb#L22?

Reply:
It was a copy-pasta when I was trying to figure out why I couldn't run the test independently; I'm not sure it's necessary tho!

Reply:
I've remofed it and things seem fine; so I think it was just for the test that dealt with mailers.